package.json

```diff
@@ -11,8 +11,8 @@
     "lint-fix": "./node_modules/.bin/eslint bin/* . --fix",
     "test": "NODE_ENV=test ./node_modules/.bin/mocha --compilers js:babel-core/register tests/helpers/*.js --require ignore-styles --recursive tests",
     "test-watch": "NODE_ENV=test ./node_modules/.bin/mocha --compilers js:babel-core/register --recursive -w tests",
-    "pree2e": "NODE_ENV=test ./scripts/pree2e.sh",
-    "e2e": "NODE_ENV=test ./node_modules/.bin/nightwatch",
+    "pree2e": "scripts/pree2e.sh",
+    "e2e": "NODE_ENV=test nightwatch",
```

Comment:
Is there a reason that we are preferring the global versus the local installation of nightwatch?